### DISCUSSION TITLE
CP2K Release 2024.2

### DISCUSSION THREAD

**Discussion by oschuett** [cp2k/cp2k#3453] [Announcements]
Hi Everybody,

following our 6 month cadence, we are planning the next release for the end of June.

So, please merge your changes in time.

**Comment by abussy** [comment]
We are hoping to have a DBCSR release by then, which addresses some of the issues observed on Alps. It would be great if we can provide CP2K v2024.2 with a way to avoid the problems reported here https://github.com/cp2k/dbcsr/issues/795

**Reply by oschuett** [reply]
I agree, that it would be good to have a DBCSR release that includes your https://github.com/cp2k/dbcsr/pull/801.

I'm also chipping away at getting DBM to feature parity, but there I'm aiming more for the 2025.1 release.

**Reply by RMeli** [reply]
Another thing we need to include in the DBCSR release is enabling H100 with CMake, re-using A100 parameters. I'll open a PR.

**Comment by RMeli** [comment]
Hi, is there already a planned release date?

**Reply by oschuett** [reply]
My plan is to wait at least until the end of June. @alazzaro how are things looking for a DBCSR release?

**Reply by alazzaro** [reply]
> My plan is to wait at least until the end of June. @alazzaro how are things looking for a DBCSR release?

Short story: can we move the release by end of July (or close to)? The current plan is to release a new RC candidate of DBCSR with new functionalities (see https://github.com/cp2k/dbcsr/pull/812). The target is the Santis support (all credits to @RMeli and @abussy ).

Long story: So, took me some time to fix some issues. The idea is to provide a version of DBCSR that @RMeli and @abussy can test on Santis (GH system). As part of that, I'm trying to enable a generic kernel that can work on any GPU. It looks like this is not an easy task (see https://storage.googleapis.com/cp2k-ci/run-cp2k-cuda-pascal-47248377_report.txt ), but I will fix it. However, Santis is offline and we are going on holidays in the next weeks... Reasonable, I will make the new RC today/tomorrow, then we can test it on Santis by mid July and include in CP2K by the week of the 22/07. 

@RMeli and @abussy please comment if needed.

**Reply by oschuett** [reply]
Yes, I think good support for Grace-Hopper is worth postponing the release by a few weeks.

**Reply by alazzaro** [reply]
New DBCSR is in.
